x-pack/plugins/entity_manager/server/lib/entities/built_in/services_from_ecs_data.ts

```diff
@@ -121,7 +128,7 @@ export const builtInServicesFromEcsEntityDefinition: EntityDefinition =
           {
             name: 'A',
             aggregation: 'doc_count',
-            filter: 'data_stream.type: logs',
+            filter: 'source_data_stream.type: logs',
```

Comment:
the filtering applies to the source data so this should remain `data_stream.type`

Reply:
https://github.com/elastic/kibana/pull/194612/commits/e43b286a3a3ec42e0424af5f30d8d49fcc3d2910